app/src/main/java/http_server/routes/HelloWorld.java

```diff
@@ -0,0 +1,7 @@
+package http_server.routes;
+
+public class HelloWorld {
+  public String getHelloWorld() {
+    return "Hello world";
+  }
+}
```

Comment:
So this is very implementation sticky in my mind, where it is specific to hello world. Would the pattern be something like SimpleResponse and the constructor takes in a String message to return in the response? 
I'm thinking more like Duck Typing. 

And instead of getHelloWorld, it would be getResponse()?